libgnucash/engine/Transaction.c

```diff
@@ -3188,14 +3188,14 @@ record_price(Split *split,
 }
 
 void
-xaccTransRecordPrice(Transaction *trans)
+xaccTransRecordPrice (Transaction *trans, PriceSource source)
 {
     /* XXX: move this into xaccSplitCommitEdit and other callers of
     * gnc_pricedb_add_price from split-register.c, gnc-imp-props-price.cpp
     * etc.
     */
    for (GList *n = xaccTransGetSplitList (trans); n; n = n->next)
-      record_price (n->data, PRICE_SOURCE_SPLIT_IMPORT);
+      record_price (n->data, source);
 }
```

Comment:
Please move both function definitions up to line 2940, above the QofObject stuff.

Reply:
sure. thanks!